src/EventTickets/ListTable/Columns/SalesAmountColumn.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\EventTickets\ListTable\Columns;
+
+use Give\EventTickets\Models\Event;
+use Give\Framework\ListTable\ModelColumn;
+use Give\Framework\Support\ValueObjects\Money;
+
+/**
+ * @unreleased
+ *
+ * @extends ModelColumn<Event>
+ */
+class SalesAmountColumn extends ModelColumn
+{
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public static function getId(): string
+    {
+        return 'sales-amount';
+    }
+
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public function getLabel(): string
+    {
+        return __('Ticket Sales', 'give');
+    }
+
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     *
+     * @param Event $model
+     */
+    public function getCellValue($model, $locale = ''): string
+    {
+        $ticketTypes = [];
+        foreach ($model->ticketTypes()->getAll() ?? [] as $ticketType) {
+            $ticketTypes[$ticketType->id] = [
+                'price' => $ticketType->price,
+                'capacity' => $ticketType->capacity,
+            ];
+        }
+
+        $salesTotal = array_reduce(
+            $model->eventTickets()->getAll() ?? [],
+            function (Money $acc, $eventTicket) use ($ticketTypes) {
```

Comment:
What does `$acc` abbreviate? Is it `accumulated`? The standard variable name for `array_reduce` is `$carry`, unless it is updated to be more specific/declarative (ie `Money $total`).

Reply:
Old habits from JS. Will change that.

Reply:
Resolved: 308e56fd30ac800972a362dd537996f9307149f3